zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -490,6 +490,18 @@ public static Authentication getAuthentication() {
         .build();
   }
 
+  public static Authentication getAnonymousAuthentication() {
+    return new Builder()
+        .token(
+            Authorization.jwtEncoder()
+                .withIssuer(JwtAuthorizationBuilder.DEFAULT_ISSUER)
+                .withAudience(JwtAuthorizationBuilder.DEFAULT_AUDIENCE)
+                .withSubject(JwtAuthorizationBuilder.DEFAULT_SUBJECT)
+                .withClaim(Authorization.AUTHORIZED_ANONYMOUS_USER, true)
+                .build())
+        .build();
```

Comment:
💭 When we implemented https://github.com/camunda/camunda/issues/23093 we don't need this JWT anymore. We could just have an `ANONYMOUS` [AuthDataFormat](https://github.com/camunda/camunda/blob/main/zeebe/protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/encoding/AuthInfo.java#L97) which we could check for in Zeebe. That makes our life a bit easier.

Reply:
That was actually my first implementation. Will #23093 be implemented with 8.7?

Reply:
I know it's on @nicpuppa his radar so it likely will be. But he's working on some more urgent issues atm.

Reply:
Yes, as @remcowesterhoud said I started working on [it](https://github.com/camunda/camunda/pull/25655). At the moment is a WIP